boards/litex/sim/README.md

```diff
@@ -0,0 +1,85 @@
+LiteX SoC in a Verilated Simulation
+============================================
+
+This board is targeting a
+[verilated](https://www.veripool.org/wiki/verilator) SoC bitstream
+built using [LiteX](https://github.com/enjoy-digital/litex).
+
+Since LiteX is a SoC builder, the individual generated bitstreams can
+differ significantly depending on the release and configuration
+options used. This board definition currently targets and has been
+tested with
+- [the LiteX SoC generator, revision
+  444a605dea](https://github.com/enjoy-digital/litex/tree/444a605deae6a561dbe2c49bf3062eae6f3cd887)
+- using the included [lx_sim.py](https://github.com/enjoy-digital/litex/blob/444a605deae6a561dbe2c49bf3062eae6f3cd887/litex/tools/litex_sim.py)
+- built around a VexRiscv-CPU
+- featuring a TIMER0 with 64-bit wide hardware uptime
+- along with the following configuration options:
+
+  ```
+  --csr-data-width=32
+  --integrated-rom-size=1048576
+  --cpu-variant=full
+  --with-ethernet
+  --rom-init $PATH_TO_TOCK_BINARY
+  ```
+
+Many bitstream customizations can be represented in the Tock board by
+simply changing the variables in `src/litex_generated.rs`. To support
+a different set of FPGA cores and perform further modifications, the
+`src/main.rs` file will have to be modified.
+
+
+Please note
+-----------
+
+This board is still in development. The memory protection (PMP)
+mechanism is not yet integrated into the VexRiscv core. Nonetheless,
+the Tock kernel works and can run multiple userspace applications.
+
+The following on-board components and cores are supported:
+- [X] Timer (with uptime support)
+- [X] UART console output
+- [X] Ethernet MAC
+
+The following components and cores require porting:
+- [ ] Memory protection (PMP) support in the VexRiscv CPU ([upstream
+      PR](https://github.com/SpinalHDL/VexRiscv/pull/147))
+
+
+Building the SoC / Programming the FPGA
+---------------------------------------
+
+Please refer to the [LiteX
+documentation](https://github.com/enjoy-digital/litex/wiki/) for
+instructions on how to install and use the LiteX SoC generator.
+
+Once LiteX is installed, running the simulation:
+
+```
+$ cd $PATH_TO_LITEX/litex/tools
+$ ./litex_sim.py <configuration options>
+```
+
+This command will build the SoC Verilog files and a ROM-bios.
+Afterwards, it will use Verilator to create a C++ simulation program,
+which is compiled and run. The script may ask for an administrator
+password to create a `tap0` network device on the host if built with
+Ethernet support.
+
+
+Debugging
+---------
```

Comment:
How do you run the kernel?

Reply:
I can add a section, but it's just `./litex_sim.py` with the configuration options stated above, so:

```
./litex_sim.py \
  --csr-data-width=32 \
  --integrated-rom-size=1048576 \
  --cpu-variant=full \
  --with-ethernet \
  --rom-init $PATH_TO_TOCK_BINARY (e.g. target/riscv32i-unknown-none-elf/release/litex_sim.bin)
```